demisto_sdk/commands/common/hook_validations/base_validator.py

```diff
@@ -225,7 +225,7 @@ def yml_has_test_key(self, test_playbooks: list, file_type: str) -> bool:
                 f'please edit the yml file and add the TestPlaybook under the \'tests\' key. '
                 f'If you don\'t want to create a'
                 f' TestPlaybook for this {file_type}, edit the yml file and add  \ntests:\n -  No tests\n lines'
-                f' to it.')
+                f' to it or run \'demisto-sdk format -i {self.file_path}\'')
```

Comment:
lets use `Errors.suggest_fix()`

Reply:
I think it will look weird with it, what do you think?

```
@staticmethod
    def suggest_fix(file_path: str, *args: List, cmd: str = 'format') -> str:
        return f'To fix the problem, try running `demisto-sdk {cmd} -i {file_path} {" ".join(args)}`'
```

Reply:
ok leave it as is